pkgs/development/libraries/gstreamer/good/default.nix

```diff
@@ -102,8 +111,9 @@ stdenv.mkDerivation rec {
     gdk-pixbuf
     aalib
     libcaca
-    libsoup
+    libsoup_3
```

Comment:
It should not be needed here, it is only checked by tests, which we do not run.

Reply:
ofborg eval kept failing with libsoup_3 removed so i added it back.